packages/react-data-grid-react-window-grid/src/components/DataGridHeaderRow/DataGridHeaderRow.types.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  DataGridContextValue,
+  DataGridRowProps,
+  DataGridRowState,
+  TableColumnDefinition,
+} from '@fluentui/react-components';
+import { FixedSizeListProps, ListChildComponentProps } from 'react-window';
+
+export type CellRenderer<TItem = unknown> = (
+  column: TableColumnDefinition<TItem>,
+  style: React.CSSProperties,
+  dataGridContextValue: DataGridContextValue,
+  /**
+   * The index of each row
+   */
+  index: number
+) => React.ReactNode;
+
+export type DataGridHeaderRowProps<TItem = unknown> = Omit<
```

Comment:
You should export:
* DataGridHeaderRowProps
* CellRenderer (this should be aliased to avoid naming conflicts)